risc0/zkvm/sdk/rust/src/host/mod.rs

```diff
@@ -15,12 +15,14 @@
 #![deny(missing_docs)]
 #![doc = include_str!("README.md")]
 
-mod exception;
+#[cfg(not(feature = "pure-prove"))]
```

Comment:
I think the way I was envisioning dealing with this was using the newly added `host` feature to prevent this entire `host` module from being compiled if you wanted the pure-rust version. But I suppose this way works in the interim just as well or perhaps better.